integration/optional-defaults-incorrect/terraform.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 1.7.0"
+  required_providers {
+    azurerm = {
+      version = ">= 3.98.0"
```

Comment:
I assume that version should be like "~> #.#" or ">= #.#, < #.#". Please also update others with same problem.